main.go

```diff
@@ -99,7 +102,14 @@ func main() {
 	metrics.InitOrFail(ctx, logger, *prometheusOn, *prometheusListenAddress, jobsGlobalConfig.ClientID, country)
 
 	reporter := newReporter(*logFormat, logger)
-	job.NewRunner(runnerConfigOptions, jobsGlobalConfig, reporter).Run(ctx, logger)
+	runner := job.NewRunner(runnerConfigOptions, jobsGlobalConfig, reporter)
+
+	if mobile.IsMobile {
```

Comment:
I would suggest to use another main_mobile.go to start a runner. In this case it even simpler to do it in separate repo and use 1000n as a dependency, as @arriven proposed before.

Reply:
so you would copy all main's content to it as well?.. imo it's not robust. Or how would you use this main's content? Is it possible to run it as a plain function inside another main?

Initially I tried having an all contained file with `init()` which runs everything without any need to modify main, but unfortunately fyne doesn't allow running its main loop not in the main thread.

Reply:
What about moving generic part of main to some API-like function, and then call it from other projects like a mobile app?

Reply:
In this case you don't need to support anything - no manifests, no stuff - just a bit of code which introduces something where to show the output. And you just build it for androids (and ios, but I didn't test that).

Anyway, even if a separate repo is to be pursued, we would need to make changes similar to turning the whole app into a "lib" and then having main.go just plug and run it. @arriven are you ok with that? would it be still preferable?